redis_enterprise_prometheus/manifest.json

```
@@ -0,0 +1,73 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f065ce60-b831-4e7d-8521-1cde67a6c12a",
+  "app_id": "redis-enterprise-prometheus",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "<FILL IN - A brief description of what this offering provides>",
+    "title": "redis_enterprise_prometheus",
+    "media": [
+      {
+        "media_type":  "image",
+        "caption": "graph of node CPU usage",
+        "image_url": "images/datadog-detail-node-cpu.png"
+      },
+      {
+        "media_type":  "image",
+        "caption": "graph of node memory usage",
+        "image_url": "images/datadog-detail-node-memory.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Data Stores",
+      "Offering::Integration",
+      "Queried Data Type::Metrics",
+      "Submitted Data Type::Metrics"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "auto_install": true,
+      "source_type_id": 44087280,
+      "source_type_name": "Redis Enterprise Prometheus",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "rdse2.",
+        "check": "rdse2.endpoint_client_connections",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    },
+    "dashboards": {
+      "redis-enterprise-prometheus-overview": "assets/dashboards/redis_enterprise-prometheus_overview.json",
+      "redis-enterprise-prometheus-database": "assets/dashboards/redis_enterprise-prometheus_database.json",
+      "redis-enterprise-prometheus-node": "assets/dashboards/redis_enterprise-prometheus_node.json",
+      "redis-enterprise-prometheus-replication": "assets/dashboards/redis_enterprise-prometheus_active-active.json",
+      "redis-enterprise-prometheus-proxy-threads": "assets/dashboards/redis_enterprise-prometheus_proxy-threads.json",
+      "redis-enterprise-prometheus-shard": "assets/dashboards/redis_enterprise-prometheus_shard.json",
+      "monitors": {},
+      "saved_views": {}
```

```suggestion
      "redis-enterprise-prometheus-shard": "assets/dashboards/redis_enterprise-prometheus_shard.json"
```

We still need to get rid of monitors and saved_views from here. It's in the wrong spot, and it's empty so we can get rid of it.